includes/templates/admin/user/date-time.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * User Date Time Settings Template.
+ *
+ * @package GatherPress\Core
+ * @since 0.29.0
+ */
+
+use GatherPress\Core\Utility;
+
+if ( ! isset( $timezone ) && ! isset( $date_format ) && ! isset( $time_format ) ) {
+	return;
+}
+
+$gatherpress_tz_choices = Utility::timezone_choices();
+
+$gatherpress_date_attrs = array(
+	'name'  => 'gp_date_format',
+	'value' => ! empty( $date_format ) ? $date_format : '',
+);
+
+$gatherpress_time_attrs = array(
+	'name'  => 'gp_time_format',
+	'value' => ! empty( $time_format ) ? $time_format : '',
+);
```

Comment:
Let's move these variables to the `profile_fields` method and out of the template. Then just pass them to the template.